imodels/rule_set/boosted_rules.py

```diff
@@ -133,13 +144,6 @@ def predict(self, X):
         X = check_array(X)
         return self.eval_weighted_rule_sum(X) > 0
 
-    # def predict(self, X):
-    #     """Predict outcome for X
-    #     """
-    #     check_is_fitted(self)
-    #     X = check_array(X)
-    #     return self.predict_proba(X).argmax(axis=1)
-
```

Comment:
Hey Sean, thanks for the contribution! Can you uncomment this instead of deleting it?

Reply:
Looks like the reason this was commented to update the predict function to use the `eval_weighted_rule_sum` function inherited from `RuleSet`. I can't uncomment because we would have conflicting definitions for the predict function. Do you want me to just leave it commented?

Reply:
I see — then deleting is ok. I think this was a reminder that we eventually want a `predict` that just takes the argmax of `predict_proba`, but we need to do some more work on `predict_proba` first. I'll add it as a todo item instead.